radio_beam/beam.py

```diff
@@ -429,6 +435,16 @@ def as_kernel(self, pixscale):
                                           self.minor.to(u.deg).value/pixscale,
                                           self.pa.to(u.radian).value)
 
+    def as_struct_element(self, pixscale):
```

Comment:
This needs a docstring.  I'm not sure about the name, either - why not `as_elliptical_kernel`, for example?